CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 
 - feat: Support storing JSON in the changes field when ```AUDITLOG_STORE_JSON_CHANGES``` is enabled.  ([#719](https://github.com/jazzband/django-auditlog/pull/719))
 - feat: Added `AUDITLOG_MASK_CALLABLE` setting to allow custom masking functions ([#725](https://github.com/jazzband/django-auditlog/pull/725))
+- Confirm Django 5.2 support.
```

Comment:
You have to move it under the `Improvements` section up bacause `3.2.0` has been released already. Also, you need to mention the PR like the other entries

`Confirm Django 5.2 support. ([#730](https://github.com/jazzband/django-auditlog/pull/730))`